Makefile

```diff
@@ -5,51 +5,52 @@ BACKEND=src/backend
 FRONTEND=src/frontend
 CHARDECODER=src/chardecoder
 BYTECOMP=$(FRONTEND)/bytecomp
-OCB_FLAGS = -cflags -w,-3 \
-	-use-ocamlfind -use-menhir \
-	-I $(SRCROOT)/ -I $(FRONTEND)/ -I $(BACKEND)/ -I $(CHARDECODER)/ -I $(BYTECOMP) \
-	-pkgs "str,ppx_deriving.show,core_kernel,uutf,batteries, \
-	menhirLib,yojson,camlimages,camlimages.jpeg,otfm,camlpdf" \
-	-tag bin_annot -tag thread -tag unsafe_string -yaccflags "--table --explain"
 TARGET=satysfi
-OCB = ocamlbuild $(OCB_FLAGS)
 BINDIR=$(PREFIX)/bin
 RM=rm -f
 RUBY=ruby
 GENCODE=./gen_code.rb
+DUNE=jbuilder
```

Comment:
[dune version 1.0.0](https://github.com/ocaml/dune/releases/tag/1.0.0) was released this week🎉, so it might be better to test against the new version and then replace here with, `DUNE=dune`.

Reply:
Thanks.

I have tried quick migration to dune 1.0.0 as in https://github.com/leque/SATySFi/tree/wip/dune-1 , but failed with the following error now.

```
File "src/dune", line 17, characters 13-36:
Error: No ppx driver were found.
Hint: Try upgrading or reinstalling ocaml-migrate-parsetree.
```

I will try to fix this error, but the migration might be appropriate to be a separated PR.

Reply:
Agreed.